change/@ni-nimble-components-788ed1e7-ebcc-469f-ab96-b5e608276b80.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We should add a note to our CONTRIBUTING documentation listing out the supported attributes (are there similar ones for Safari / Chrome / Edge?) and describing our policy for when it's appropriate to use them (pending the discussion in Milan's thread to decide what that policy is).

Reply:
Added